website/docs/sql/scalar_func/datetime.md

```diff
@@ -476,3 +524,45 @@ tuple will be used, and not the timestamp of table creation. This works
 differently in PostgreSQL, where the table creation timestamp is used in
 every insertion that uses the default value.
 
+## Fiscal Options {#fiscal-options}
+
+The fiscal options can be used in [EXTRACT](#datetime-extract),
+[date_trunc](#datetime-trunc) and [date_part](#datetime-datepart) functions,
+when the field for those functions is one of the fiscal fields:
```

Comment:
add explanation on what those options do on a high-level, i.e., "define the start date of the fiscal calendar" and in which cases to use them